### PR TITLE
[Feature] System Health sidepanels upgrade to full page

### PR DESCRIPTION
**Pull request by daiyiwhale**
System Health side panels get an upgrade to full page ✨

depends on: 
- https://github.com/dcos/dcos-ui/commit/45908c02b13614f82ca5d3032b1f741bd85f9747 for correctly inverse styled dropdown
- https://github.com/dcos/dcos-ui/pull/82 for headers

notes:
- The top tabs are to be removed in detail views in the future, when the breadcrumbs are introduced.

`/system/overview/components`

`/system/overview/components/<component>`

`/system/overview/components/<component>/nodes/<node>`

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @philipnrmn

File: src/js/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    );
+  }
+}
+
+Header.defaultProps = {
+  className: 'container container-fluid container-pod container-pod-short flush flush-top',
+  mediaWrapperClassName: 'media-object-spacing-wrapper media-object-spacing-narrow media-object-offset',
+  dividerClassName: 'container-pod container-pod-short flush-top container-pod-divider-bottom container-pod-divider-bottom-align-right container-pod-divider-inverse',
+  titleClassName: 'flush inverse',
+  iconClassName: 'icon icon-large icon-image-container icon-app-container',
```
Comment (philipnrmn) [inline src/js/components/Header.js]: How come we need so much control over the internal class structure? Can't we control internal style using classes on the wrapper alone?

Comment (mlunoe) [inline src/js/components/Header.js]: We have been burnt in the past by not being able to have control over classes

Comment (philipnrmn) [inline src/js/components/Header.js]: I just realised this PR is the wrong place for this comment anyway - I should have put it on https://github.com/dcos/dcos-ui/pull/82. Plz ignore :)

File: src/js/components/TaskView.js [hunk 2]
```diff
@@ -159,6 +167,18 @@ class TaskView extends mixin(SaveStateMixin) {
   getContent() {
     let state = this.state;
     let tasks = this.props.tasks || [];
+    let headerClassSet = classNames({
```
Comment (philipnrmn) [inline src/js/components/TaskView.js]: `classNames` needs to be imported here.

**Comment by philipnrmn** [comment]
Do we want tests around the new `UnitsHealthDetail` component? Otherwise, LGTM 👍

File: src/js/components/UnitHealthNodesTable.js [hunk 3]
```diff
@@ -82,23 +83,13 @@ class UnitHealthNodesTable extends React.Component {
...
-    let parentPaths = this.props.parentRouter.getCurrentRoutes();
-    let parentPath = parentPaths[parentPaths.length - 2].name;
-    let path = 'system-overview-units-unit-nodes-node-panel';
-    // Route is available under dashboard root
-    if (parentPath === 'dashboard') {
-      path = 'dashboard-units-unit-nodes-node-panel';
-    }
-
-    let params = {unitID: this.props.itemID, unitNodeID: node.get('host_ip')};
+    let params = Object.assign(this.props.params, {unitNodeID: node.get('host_ip')});
```
Comment (kennyt) [inline src/js/components/UnitHealthNodesTable.js]: can we put an empty object as the first argument of assign?

**Comment by daiyiwhale** [comment]
@philipnrmn the tests for this component are integration tests!

**Comment by daiyiwhale** [comment]
Alright we are rebased and ready to go 👍

**Comment by daiyiwhale** [comment]
Rebased to new hot reloading build for your convenience ✨  ✨  ✨

**Pull request merged**